docs/usage/request_cert.md

```diff
@@ -133,6 +135,10 @@ spec:
   # the DNS challenge will be written to '_acme-challenge.writable.domain.com'.
   #followCNAME: true
 
+  # optionally set labels for the secret
+  #secretLabels:
+  #  key1: value1
+  #  key2: value2
 ```
 
 ## Supported attributes
```

Comment:
Shouldn't the `secretLabels` also be added to the list below?

Reply:
Fixed, thanks!